### PR TITLE
[enhance](mtmv)show create materialized view

### PR DESCRIPTION
**Pull request by zddr**
- support show create materialized view
- Prohibit show create table from specifying MTMV
  
```
 mysql> show create MATERIALIZED VIEW mv1\G
*************************** 1. row ***************************
       Materialized View: mv1
Create Materialized View: CREATE MATERIALIZED VIEW mv1
(k1,k2)
BUILD DEFERRED REFRESH AUTO ON MANUAL
DUPLICATE KEY(`k1`, `k2`)
DISTRIBUTED BY RANDOM BUCKETS 2
PROPERTIES (
"replication_allocation" = "tag.location.default: 1",
"min_load_replica_num" = "-1",
"is_being_synced" = "false",
"storage_medium" = "hdd",
"storage_format" = "V2",
"inverted_index_storage_format" = "V2",
"light_schema_change" = "true",
"disable_auto_compaction" = "false",
"enable_single_replica_compaction" = "false",
"group_commit_interval_ms" = "10000",
"group_commit_data_bytes" = "134217728"
)
AS SELECT * from user1
1 row in set (0.01 sec)
  ```
```
mysql> show create table rollup_mv;
ERROR 1105 (HY000): errCode = 2, detailMessage = not support async materialized view, please use `show create materialized view`
```
doc: https://github.com/apache/doris-website/pull/870

### CODE REVIEW

File: fe/fe-core/src/main/cup/sql_parser.cup [hunk 1]
```diff
@@ -4278,6 +4278,10 @@ show_param ::=
     {:
         RESULT = new ShowCreateTableStmt(table, true, false);
     :}
+    | KW_CREATE KW_MATERIALIZED KW_VIEW table_name:table
+    {:
+        RESULT = new ShowCreateMTMVStmt(table);
+    :}
```
Comment (morrySnow) [inline fe/fe-core/src/main/cup/sql_parser.cup:4284]: impl it in Nereids?

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run performance

**Review by seawinde — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**